stellar-core-prometheus-exporter/debian/stellar-core-prometheus-exporter.service

```diff
@@ -1,21 +1,14 @@
 [Unit]
 Description=SDF - stellar-core-prometheus-exporter
-Before=multi-user.target
-#PartOf=stellar.service
 
 [Service]
-User=stellar
```

Comment:
What is the reasoning behind removing `User=stellar` and `Group=stellar` ?

Reply:
I realized that there is no guarantee that the stellar user and group exists on the host running the exporter. Defaulting to root avoids potential problems.

I can think of two situations where the stellar user does not exist:
1. The exporter running on a different host than stellar-core. We default to scraping 127.0.0.1 but users can monitor remote hosts as well
2. In k8s world users might run the exporter as separate container in the stellar-core pod. In this case they will scrape 127.0.0.1 since containers in one pod share network namespace. However there is no guarantee that the monitoring container has the stellar user.

Reply:
Thanks @jacekn, that sounds good.